app/jsx/dashboard_card/DashboardColorPicker.js

```diff
@@ -121,14 +121,14 @@ import cx from 'classnames'
 
     leftPosition: function(){
       return this.tooltipOnRight() ?
-        (this.leftPlusElement() - 80) :
+        (this.leftPlusElement()) :
         (this.leftPlusElement() - 360)
     },
 
     pickerToolTipStyle: function() {
       if (this.props.isOpen) {
         return {
-          position: 'absolute',
+          position: 'fixed',
```

Comment:
this fixes a problem from outside of this branches intent; lets carry on with this particular one, but in the future, for small bug fixes, lets make tiny branches so we can merge back in fixes early and often.